api/views/phones.py

```diff
@@ -458,6 +458,8 @@ def _check_and_update_contact(inbound_contact, contact_type):
         raise exceptions.ValidationError(f"Number is not accepting {contact_type}.")
 
     inbound_contact.last_inbound_date = datetime.now(timezone.utc)
+    # strip trailing "s": InboundContact.last_inbound_type is max_length 4
+    inbound_contact.last_inbound_type = contact_type.strip("s")
```

Comment:
```suggestion
    inbound_contact.last_inbound_type = contact_type[:-1]
```
to drop the last character in case we deal with future types like `"spams"` which the current logic will return `"pam"` (unsure if this is a real contact type but was a delicious salty example).